[bluetooth.bluez] Exception when pausing Thing

```text
2026-03-22 13:11:23.780 [DEBUG] [th.bluez.internal.BlueZBridgeHandler] - Termination of DBus BlueZ handler
2026-03-22 13:11:23.781 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.dispose()' on 'org.openhab.binding.bluetooth.bluez.internal.BlueZBridgeHandler@419fe096': No discovery started
org.freedesktop.dbus.exceptions.DBusExecutionException: No discovery started
	at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(Unknown Source) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:?]
	at org.freedesktop.dbus.messages.Error.getException(Error.java:113) ~[?:?]
	at org.freedesktop.dbus.messages.Error.throwException(Error.java:138) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:241) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:254) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:156) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:99) ~[?:?]
	at jdk.proxy44.$Proxy440.StopDiscovery(Unknown Source) ~[?:?]
	at com.github.hypfvieh.bluetooth.wrapper.BluetoothAdapter.stopDiscovery(BluetoothAdapter.java:282) ~[?:?]
	at org.openhab.binding.bluetooth.bluez.internal.BlueZBridgeHandler.dispose(BlueZBridgeHandler.java:120) ~[?:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:149) ~[?:?]
	at org.openhab.core.internal.common.Invocation.call(Invocation.java:52) ~[?:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
2026-03-22 13:11:23.787 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while disposing handler of thing 'bluetooth:bluez:hci0': No discovery started
org.freedesktop.dbus.exceptions.DBusExecutionException: No discovery started
	at jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(Unknown Source) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:?]
	at org.freedesktop.dbus.messages.Error.getException(Error.java:113) ~[?:?]
	at org.freedesktop.dbus.messages.Error.throwException(Error.java:138) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:241) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:254) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.executeRemoteMethod(RemoteInvocationHandler.java:156) ~[?:?]
	at org.freedesktop.dbus.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:99) ~[?:?]
	at jdk.proxy44.$Proxy440.StopDiscovery(Unknown Source) ~[?:?]
	at com.github.hypfvieh.bluetooth.wrapper.BluetoothAdapter.stopDiscovery(BluetoothAdapter.java:282) ~[?:?]
	at org.openhab.binding.bluetooth.bluez.internal.BlueZBridgeHandler.dispose(BlueZBridgeHandler.java:120) ~[?:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at org.openhab.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:149) ~[?:?]
	at org.openhab.core.internal.common.Invocation.call(Invocation.java:52) ~[?:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen -->
<!-- If you're suggesting a change/improvement, tell us how it should work -->

No exception.

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->

## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1.
2.

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->
<!-- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 5.1.3
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): Linux openhabian 6.12.47+rpt-rpi-2712 1 SMP PREEMPT Debian 1:6.12.47-1+rpt1~bookworm (2025-09-16) aarch64 GNU/Linux

--- a/bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/BlueZBridgeHandler.java
+++ b/bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/BlueZBridgeHandler.java
@@ -111,13 +111,17 @@ public class BlueZBridgeHandler extends AbstractBluetoothBridgeHandler<BlueZBlue
 
         Future<?> job = discoveryJob;
         if (job != null) {
-            job.cancel(false);
+            job.cancel(true);
             discoveryJob = null;
         }
 
         BluetoothAdapter adapter = this.adapter;
         if (adapter != null) {
-            adapter.stopDiscovery();
+            try {
+                adapter.stopDiscovery();
+            } catch (DBusExecutionException e) {
+                logger.debug("Failed to stop discovery on adapter during disposal", e);
+            }
             this.adapter = null;
         }
 

